core/templates/dev/head/pages/Base.js

```diff
@@ -23,6 +23,7 @@ oppia.controller('Base', [
       SidebarStatusService) {
     $scope.AlertsService = AlertsService;
     $scope.currentLang = 'en';
+    $scope.siteFeedBackFormURL = GLOBALS.SITE_FEEDBACK_FORM_URL;
```

Comment:
Note: "feedback" is one word

Reply:
Got it!
Changing to `$scope.siteFeedbackFormURL`